ietf/templates/submit/announce_to_lists.txt

```diff
@@ -1,26 +1,27 @@
-{% autoescape off %}
-{% filter wordwrap:78 %}A New Internet-Draft is available from the on-line Internet-Drafts directories.{% if submission.group %} This Internet-Draft is a work item of the {{ submission.group.name }} ({{ submission.group.acronym|upper }}){% if submission.group.type.name %} {{ submission.group.type.name }}{% endif %} of the {% if submission.group.type_id == "rg" %}IRTF{% else %}IETF{% endif %}.{% endif %}{% endfilter %}
+{% autoescape off %}{% filter wordwrap:78 %}Internet-Draft {{ submission.name }}-{{ submission.rev }}.txt is now available.{% if submission.group %} It is a work item of the {{ submission.group.name }} ({{ submission.group.acronym|upper }}){% if submission.group.type.name %} {{ submission.group.type.name }}{% endif %} of the {% if submission.group.type_id == "rg" %}IRTF{% else %}IETF{% endif %}.{% endif %}{% endfilter %}
 
-   Title           : {{ submission.title }}
-   Author{{ submission.authors|pluralize:" ,s" }}         : {% for author in submission.authors %}{{ author.name }}{% if not forloop.last %}
-                     {% endif %}{% endfor %}
-   Filename        : {{ submission.name }}-{{ submission.rev }}.txt
-   Pages           : {{ submission.pages }}
-   Date            : {{ submission.submission_date|date:"Y-m-d" }}
+   Title:   {{ submission.title }}
+   Author{{ submission.authors|pluralize:",s" }}: {% if submission.authors|length == 1 %} {% endif %}{% for author in submission.authors %}{{ author.name }}{% if not forloop.last %}
```

Comment:
Why the extra space here when there is only one author?

Reply:
Because with multiple authors the string before the colon is `Authors`, i.e., one character longer.

Reply:
Consider the discussion of pluralizing cherry at https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#pluralize